pom.xml

```diff
@@ -207,6 +207,12 @@
         <artifactId>commons-net</artifactId>
         <version>1.4.1</version>
       </dependency>
+      <dependency>
+        <groupId>commons-dbcp</groupId>
+        <artifactId>commons-dbcp</artifactId>
+        <version>1.4</version>
+        <scope>provided</scope>
+      </dependency>
```

Comment:
dbcp is niet nodig als dependency. Kan hier verwijderd worden.

Reply:
Gedaan.